hugo/layouts/index.html

```diff
@@ -61,169 +34,238 @@ <h4 class="text-dark m-0">
+                                <span class="text-medium">At source{d} we are creating a suite of Open Source tools enabling "Code as Data" and "Machine Learning on Code". We are also great believers in Open Source and its philosophy.</span>
+                            </div>
+                        </a>
                     </div>
                 </div>
-            </div>
-            <div class="col-sm-6 mb-5 mb-sm-4">
-                <div class="card text-white orange h-100">
-                    <div class="card-body p-4 p-lg-5">
-                        <img src="img/icons/placeholder4.svg" alt="">
-                        <p class="card-intro pt-5 mb-0">source{d} Lookout</p>
-                        <h3 class="card-title">Machine Learning on Code</h3>
-                        <p class="card-text">Distill all the knowledge from your code base to assist engineering teams to review code faster while missing fewer mistakes.</p>
-                        <p><strong>Features</strong></p>
-                        <ul class="small-list">
-                            <li>Inferred style guides</li>
-                            <li>Naming suggestions</li>
-                            <li>Common bug detection</li>
-                            <li>Performance bottlenecks identification</li>
-                            <li>Security vulnerabilities</li>
-                            <li>Suspicious code indicators</li>
-                            <li>Language agnostic assisted coding</li>
-                        </ul>
-                        <p class="mt-5"><strong>Use cases</strong></p>
-                        <ul class="small-list">
-                            <li>Code reviewer attention guide</li>
-                            <li>Comment suggestion</li>
-                            <li>Author pre-check tool</li>
-                        </ul>
-                        <a href="lookout" class="btn btn-secondary-light w-100 mt-4">Learn More</a>
+                <div class="col-sm-12 col-md-4 mb-4">
+                    <div class="card white h-100">
+                        <a href="community">
+                            <img class="w-100" src="img/mloncode.jpg" alt="Open Source">
+                            <div class="card-body p-5">
+                                <h4 class="text-dark title-line line-yellow">MLonCode Community</h4>
+                                <span class="text-medium">Connect with other MLonCode enthusiasts and researchers through events and online resources! From our slack community to our paper reading club and meetups, there is something for everyone no matter where you're located. </span>
+                            </div>
+                        </a>
+                    </div>
+                </div>
+                <div class="col-sm-12 col-md-4 mb-4">
+                    <div class="card white h-100">
+                        <a href="community">
```

Comment:
It points to the very same point than MLonCode card, and both are the `community` page.
I wonder if we should link to different pages or sections.